fhir-path/src/main/java/com/ibm/fhir/path/FHIRPathTimeValue.java

```diff
@@ -74,7 +76,7 @@ public LocalTime time() {
      *     a new FHIRPathTimeValue instance
      */
     public static FHIRPathTimeValue timeValue(String text) {
-        LocalTime time = LocalTime.parse(text, PARSER_FORMATTER);
+        LocalTime time = ModelSupport.truncateTime(LocalTime.parse(text, PARSER_FORMATTER), ChronoUnit.MICROS);
```

Comment:
Is this automatically appending microseconds?

Reply:
Well, if a LocalTime instance is created without specifying any fractional seconds, the fields within that instance representing the fractional seconds will just be set to 0 (MILLI_OF_SECOND, MICRO_OF_SECOND, NANO_OF_SECOND), so in that sense, every TemporalAccessor always has a value for microseconds. The ModelSupport.truncateTime() method is calling the LocalTime.truncatedTo() method, which simply sets the fields within the LocalTime instance that are smaller than the specified unit to zero. So in this case, it will set the NANO_OF_SECOND field to 0, effectively truncating any precision greater than microseconds.